tutorial/template-tutorial.md

```diff
@@ -1,45 +1,87 @@
----
-title: template for tutorial # Describe the title of your article by replacing "template for tutorial" with the page name you want to publish in Confluence.
-author: The Good Docs Project
-uri-repo: https://github.com/projectname
-company: REPLACEME
-product: # Set the product parameter to add the product name here.
-app: # Insert the name of the app here.
-tags: # Set other global keywords here like app name and product name or any other likely labels. These are comma-separated tags.
----
-{Article variables (delete the comments and add in the values) }
-{Begin your Reference topic in this section. 
-For help with writing and structuring a reference article, see the README.md in the template directory for basic guidelines and links.
-Check out https://www.markdownguide.org/basic-syntax/ if you get stuck with AsciiDoc syntax.}
+# Title
 
-# Goal
-Summarise what this tutorial is about in a sentence or two. What you put here is reused in the Goal section and included in HTML description tags.
+{Before using this template, read the accompanying [tutorial template guide](about-tutorial.md).}
 
-# Before you start
-{Delete this section if your readers can dive straight into the lesson without requiring any prerequisite knowledge.}
-Make sure you meet the following prerequisites before starting the tutorial steps:
+## Overview
 
-* Prerequisite one
-* Prerequisite two
-* etc.
+In this tutorial, you'll learn how to {learning objective} using {product}. This tutorial is intended for {audience}. It assumes that you have basic knowledge of {concept 1}, {concept 2}, and {concept 3}. 
```

Comment:
I'm tentitively in favor of this if used sparingly. I could see how this advice could be over-used and become a wall of text. It might be worth describing the trade-off of adding extra description vs value of concise.